src/hooks/useGoalResource.ts

```diff
@@ -1,49 +1,446 @@
-import { useCallback } from 'react';
+import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import { GoalAchieveCriteria } from '@prisma/client';
 
 import { trpc } from '../utils/trpcClient';
 import { notifyPromise } from '../utils/notifyPromise';
+import { GoalStateChangeSchema, GoalUpdate, ToggleGoalDependency } from '../schema/goal';
+import { GoalByIdReturnType } from '../../trpc/inferredTypes';
+import {
+    AddCriteriaScheme,
```

Comment:
Could u create issue to fix them all? It mast ends with `a` letter not `e`. Thanks!

Reply:
https://github.com/orgs/taskany-inc/projects/3/views/2?pane=issue&itemId=39301212